src/app/fundraisings/[fundraisingId]/jars/JarsList/AddJarDialog/AddJarDialog.tsx

```diff
@@ -23,27 +23,30 @@ export const AddJarDialog = ({
   addJar,
   jars,
   buttonClassName,
+  fundraisingId,
 }: {
-  addJar: JarsPageState['addJar'];
+  addJar: JarsPageState["addJar"];
   jars: Array<Jar>;
   buttonClassName: string;
+  fundraisingId: string;
 }) => {
   const formRef = useRef<HTMLFormElement>(null);
 
-  const [errorText, setErrorText] = useState('');
+  const [errorText, setErrorText] = useState("");
 
   const resetForm = () => {
     formRef.current?.reset();
-    setErrorText('');
+    setErrorText("");
   };
 
   const { openDialog, dialogState, closeDialog } = useDialog({
     prepareClosing: resetForm,
   });
 
   const handleSubmit = async (formData: FormData) => {
-    const url = formData.get('url') as string;
-    const owner = formData.get('ownerName') as string;
+    const url = formData.get("url") as string;
+    const owner = formData.get("ownerName") as string;
+    const color = formData.get("jarColor") as string;
 
     const existingJar = jars.find((jar) => {
```

Comment:
I think you need similar validation for situation when user selected existing color.

Reply:
I will add it later because I plan to redo the color selection UI.